app/src/main/res/drawable/finger_tapping_complete.xml

```diff
@@ -0,0 +1,45 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Lets move the motor-control and RS2 specific resources to the motor-control module. Things like the ic_launcher can stay